pages/popover/text-align.page.tsx

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import ScreenshotArea from '../utils/screenshot-area';
+import React from 'react';
+import Box from '~components/box';
+import Popover from '~components/popover';
+
+export default function () {
+  const content = 'Nothing to see here';
+  return (
+    <ScreenshotArea>
+      <Box variant="p">
+        Very <Popover content={content}>important</Popover> message.
```

Comment:
We did not have a screenshot test on "popover inside text" scenario, so I added one as well